src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationResponse.cs

```diff
@@ -1,11 +1,12 @@
 using System.Collections.Generic;
+using VirtoCommerce.ExperienceApiModule.Core.Models;
 
 namespace VirtoCommerce.ExperienceApiModule.Core.Queries
 {
     public class PasswordValidationResponse
     {
         public bool Succeeded { get; set; }
 
-        public IList<string> ErrorCodes { get; set; }
+        public IList<IdentityErrorInfo> Errors { get; set; }
```

Comment:
Avoid breaking changes.
```suggestion
        [Obsolete("User Errors insted")]
        public IList<string> ErrorCodes { get; set; }
        public IList<IdentityErrorInfo> Errors { get; set; }
```

Reply:
Not a breaking change.